src/Doctrine/Annotation/Tokens.php

```diff
@@ -14,8 +14,7 @@
 
 namespace PhpCsFixer\Doctrine\Annotation;
 
-use Doctrine\Common\Annotations\DocLexer;
-use Doctrine\Common\Lexer\Token as LexerToken;
+use PhpCsFixer\Doctrine\Annotation\Token as DoctrineToken;
```

Comment:
```suggestion
use PhpCsFixer\Doctrine\Annotation\Token as AnnotationToken;
```
I believe it would show the intent better. Doctrine is an implementation detail here, the important part is that is a token representing phpDoc annotation.

Reply:
Done, I haven't commit this because the change required update in 2 places: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7263/commits/3c1ac14d1059e440b97cb8d4d901b13c064b5cb8